File: lib/features/cerca/domain/entities/job_offer.dart
/// A bid received from a technician during the "licitación" flow.
class JobOffer {
  const JobOffer({
    required this.id,
    required this.name,
    required this.mono,
    required this.oficio,
    required this.rating,
    required this.price,
    required this.eta,
    required this.verified,
  });

  final int id;
  final String name;
  final String mono;
  final String oficio;
  final double rating;
  final int price;
  final String eta;
  final bool verified;
}

File: backend/src/features/job/domain/job_offer.rs
use uuid::Uuid;

#[derive(Debug, Clone)]
pub struct JobOffer {
    pub id: i32,
    pub job_request_id: Uuid,
    pub technician_profile_id: i32,
    pub price: i32,
    pub eta: String,
    pub name: String,     // Joined from user name
    pub mono: String,     // Initials
    pub oficio: String,   // Technician specialty
    pub rating: f64,      // Technician rating
    pub verified: bool,   // Technician verification status
}
